src/components/Messages/MessageList/livechatCall.js

```diff
@@ -0,0 +1,60 @@
+import { h } from 'preact';
+import { useState } from 'preact/compat';
+
+import { Avatar } from '../../Avatar';
+import { Button } from '../../Button';
+import { Screen } from '../../Screen';
+import { createClassName, getAvatarUrl } from '../../helpers';
+import { parseDate } from '../MessageTime/index';
+import styles from './styles.scss';
+
+// get roomid(GetRoomId(rid)) and put rid id in iframe link
+// <iframe style="height:80%" src="localhost:3000/livechat/roomId"></iframe>
+// For example, for now I have add link of jitsi and its working fine, similarly we can add link of our video/audio call implemented with webrtc.
+const GetRoomId = (rid) => (
```

Comment:
:man_shrugging: The function name is `GetRoomId` but its functionality is to display the iframe :confused: ?